.github/workflows/constraint_tests.yml

```
@@ -0,0 +1,45 @@
+
+      - name: Install System Dependencies
+        run: |
+          sudo apt-get update
+          sudo apt install python3-dev swig libssl-dev portaudio19-dev libpulse-dev libfann-dev
+
+      - name: Download constraints file
+        run: |
+          curl -o requirements/constraints-${{ matrix.constraints_file }}.txt https://raw.githubusercontent.com/OpenVoiceOS/ovos-releases/refs/heads/main/constraints-${{ matrix.constraints_file }}.txt
+
+      - name: Install core repo with constraints
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2102:info:1:17: Ranges can only match single chars (mentioned due to duplicates)

(shellcheck)